### PR TITLE
Add new Music Lab string

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR adds a new Music Lab string that will be used for the Music Lab [play view (WIP)](https://github.com/code-dot-org/code-dot-org/pull/58068). Due to the localization deadline, we're adding this in a separate PR.

[Slack thread discussion about string](https://codedotorg.slack.com/archives/C04TH8400AU/p1714061422202479)

## Warning!!

The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) is in progress. The most critical dates are from April 3 - April 30, 2024. Please consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking anything in these two labs. Even small changes, such as a different button color, are considered significant during this time period. Reach out to the Student Learning team or Curriculum team for more details.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by bencodeorg — APPROVED** [review]

**Review by thomasoniii — APPROVED** [review]
never am I tempted more to write paragraphs of text on an approval than when it's a simple quick change. 😉 

But I'll behave again this time.

lgtm 🚀

File: apps/i18n/music/en_us.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
   "blockly_toolboxCategorySimple": "Simple",
   "blockly_toolboxCategoryTracks": "Tracks",
   "blockly_toolboxCategoryVariables": "Variables",
+  "buildWithMusicLab": "Build with Music Lab",
```
Comment (cnbrenci) [inline apps/i18n/music/en_us.json]: was this supposed to be `Built`? (jw based on slack convo this morning)

Comment (fisher-alice) [reply apps/i18n/music/en_us.json]: 🙏  - if I could, I'd give you a very big hug right now.

**Review by sanchitmalhotra126 — APPROVED** [review]

**Review by cnbrenci — APPROVED** [review]
🥳 🚀

**Pull request merged**